backend/lucene/src/main/java/org/hibernate/search/backend/lucene/search/query/impl/LuceneSearchQueryImpl.java

```diff
@@ -148,6 +153,26 @@ public Sort luceneSort() {
 		return luceneSort;
 	}
 
+	SearchScrollResult<H> next(HibernateSearchMultiReader indexReader, Integer offset, Integer limit) {
+		timeoutManager.start();
+
+		ReadWork<LuceneLoadableSearchResult<H>> work = workFactory.search( searcher, offset, limit );
```

Comment:
As far as I can tell, you didn't re-implement the exponentially-increasing behavior of Search 5. Is there a specific reason? I'm worried this will result in bad performance. For 100 pages of 100 elements, you will execute the query 100 times, while the Search 5 solution would just execute it 5 times...

Reply:
I simply forgot to do that. I'll do it in a different commit.